debugging/coreclr/debuggerClient.ts

```diff
@@ -19,7 +19,7 @@ export class DefaultDebuggerClient {
 
     // This script determines the "type" of Linux release (e.g. 'alpine', 'debian', etc.).
     // NOTE: The result may contain line endings.
-    private static debuggerLinuxReleaseIdScript: string = '/bin/sh -c \'ID=default; if [ -e /etc/os-release ]; then . /etc/os-release; fi; echo $ID\'';
+    private static debuggerLinuxReleaseIdScript: string = '/bin/sh -c "ID=default; if [ -e /etc/os-release ]; then . /etc/os-release; fi; echo $ID"';
```

Comment:
Not checked on Linux or OS X machines.

Reply:
Unfortunately, double-quotes only works on Windows; Mac and Linux require single quotes (else the interpretation is performed by the host machine rather than the container).  I've created a separate PR (#798) with just this fix (to disambiguate it from your F# changes) and updated it to work across the platforms.

Reply:
Great! I will remove this commit from the branch.